content/millennium/dstu2/individuals/patient.md

```diff
@@ -145,6 +149,88 @@ The ability to perform patient combine or uncombine operations is not available
 
 The common [errors] may be returned.
 
+## Create
+
+Create an individual Patient:
+
+    POST /Patient
+
+_Implementation Notes_
+
+* The following elements are not supported and will cause an error to be returned if set:
+    * [Patient.deceased]
+    * [Patient.multipleBirth]
+    * [Patient.photo]
+    * [Patient.contact]
+    * [Patient.animal]
+    * [Patient.managingOrganization]
+    * [Patient.link]
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
+
+### Body fields
```

Comment:
I know we have lowercase 'fields' elsewhere on the site, but this is a title and should be uppercase. Not sure if consistency or correctness is more aesthetically pleasing

Reply:
Hmm....I'd prefer to go for correctness I think. I'll update this to Fields